src/less/avatar/avatar.less

```diff
@@ -1,10 +1,28 @@
 @import "../variables/variables.less";
 
+// These are temporary colors until we get the new palette
+@_avatar-teal-foreground-color: #002a69;
+@_avatar-teal-background-color: #84b4fb;
+@_avatar-light-teal-foreground-color: #07465a;
+@_avatar-light-teal-background-color: #44ccd5;
+@_avatar-green-foreground-color: #1b561a;
+@_avatar-green-background-color: #4ce160;
+@_avatar-lime-foreground-color: #4e4e0c;
+@_avatar-lime-background-color: #c1d737;
+@_avatar-yellow-foreground-color: #553b06;
+@_avatar-yellow-background-color: #ffbd14;
+@_avatar-orange-foreground-color: #562501;
+@_avatar-orange-background-color: #ff8806;
+@_avatar-pink-foreground-color: #4b112d;
+@_avatar-pink-background-color: #f79cc8;
+@_avatar-magenta-foreground-color: #3e135f;
+@_avatar-magenta-background-color: #cc9ef0;
+
 .avatar {
     align-items: center;
-    background-color: var(--color-marketing-teal-background-5);
+    background-color: @_avatar-teal-background-color;
```

Comment:
Couldn't we just replace the current colors of the `css` props and then replace those colors again when we get more permanent colors? That wouldn't be breaking. Why move these to `less`? And if we _do_ want these to live in `less`, shouldn't we remove the `css` props? Maybe it depends on what this transition is supposed to look like and what the end result is supposed to look like.

Reply:
Those variables could be used, especially since they are existing variables in the design system. Ideally we want to keep those colors around until we replace them with the new names. I didn't change the colors because those variables could still be used by teams and they could expect the old colors. 
I moved them to less for a temporary change, like we have in other places (see EEK). Ideally if these are temporary variables until we get permanent changes. I also did not want to add other css variables that will be removed at a later time since they are populated in the client code. 
Once we get the new variables, we will update accordingly. This is a temporary change.

Reply:
Sounds good.